source/sync/configure/enable-sync.txt

```diff
@@ -41,23 +41,16 @@ If you are using the older Partition-Based Sync mode, refer to
 Prerequisites
 -------------
 
-While you are configuring Device Sync, you must define the data access patterns
-and rules for your App. If you haven't already decided how you want to
-configure your data model and access Device Sync, see:
-
-- :ref:`Configure Your Data Model <sync-schema-overview>`
-- :ref:`Sync Rules and Permissions <sync-permissions>`
-
-.. important::
-   
-   Before defining sync rules and enabling sync, you must specify at least
-   one valid :ref:`schema <schemas>` for a collection in the synced cluster 
-   unless you are using :ref:`Development Mode <development-mode>`.
-   
-   At a minimum, the schema must define ``_id`` and the field(s)
-   you intend to use as your :ref:`queryable fields <queryable-fields>`.
-   
-   For more details on how to define a schema, see :ref:`enforce-a-schema`.
+If you haven't already decided how you want to configure your data model, see
+:ref:`sync-schema-overview`.
+
+Unless you are using :ref:`Development Mode <development-mode>`, you must
+specify at least one valid :ref:`schema <schemas>` for a collection in the
+synced cluster before enabling Sync. At a minimum, the schema must define
+``_id`` as a primary key. You should also include the field(s) you intend to use
```

Comment:
Not relevant to rules, but as an FYI queryable fields being required is actually changing right now (this is what the new UI looks like):

![Screen Shot 2023-02-10 at 1 47 50 PM](https://user-images.githubusercontent.com/37843399/218173476-e97e6524-42ae-4ed1-b0b4-37943559b8fc.png)

The tl;dr is queryable fields are becoming an optional step for flexible sync; instead incoming client queries will automatically have fields referenced in the query added to the sync metadata behind-the-scenes. See https://jira.mongodb.org/browse/BAAS-18399 for more information (cc @sean-brandenburg)

Reply:
Thanks for calling this out. I've also filed [DOCSP-27844](https://jira.mongodb.org/browse/DOCSP-27844) for updating the relevant documentation for this change

Reply:
Oh cool. Feel free to ignore this then

Reply:
(feel free to resolve this)